liquibase-standard/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -274,6 +275,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDescription("If true 'classpath:' prefix will be preserved in normalized paths, allowing to resolve hierarchical resources under a classpath-based root.")
                 .setDefaultValue(false)
                 .build();
+
+        ALLOW_INHERIT_LOGICAL_FILE_PATH = builder.define("allowInheritLogicalFilePath", Boolean.class)
+                .setDescription("If true, included changelogs without an explicit logicalFilePath will inherit their parent changelog's logicalFilePath (Liquibase 4.31.0+ behavior). If false, included changelogs use their physical file paths (corrected behavior that prevents unintended inheritance). Defaults to true for backward compatibility.")
+                .setDefaultValue(true)
```

Comment:
Wouldn't the default value need to be false to be backwards compatible with versions < 4.31.0?